screed/fasta.py

```diff
@@ -23,7 +23,7 @@ def fasta_iter(handle, parse_description=False, line=None):
 
         line = to_str(line.strip())
         if not line.startswith('>'):
-            raise IOError("Bad FASTA format: no '>' at beginning of line")
+            raise IOError("Bad FASTA format: no '>' at beginning of line: {}".format(line))
```

Comment:
To further cement our commitment to not support pre-3.6 Python versions:
```suggestion
            raise IOError(f"Bad FASTA format: no '>' at beginning of line: {line}")
```